src/MahApps.Metro/MahApps.Metro.Shared/Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs

```diff
@@ -162,6 +167,22 @@ public bool IsEnabled
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that specifies label to display.
```

Comment:
`Gets or sets a value that specifies tooltip to display.`

Reply:
Completed in latest commit